### DISCUSSION TITLE
https://www.yunyitang.me/about/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#24] [Comments]
# [关于我](https://www.yunyitang.me/about/)

虽然现在是独立博客式微的时代，但无论你是循关键词自搜索引擎而来，还是从 SNS、朋友引荐而来，能在众多博客中与你相遇，也未尝不是一种缘分。欢迎你！

YUNYI BLOG 是我的个人独立博客，基本上啥都写。原创内容多为技术分享、踩坑记录与日常吐槽，以“有用且有趣”为追求，偶尔也会写点正经文字。

https://www.yunyitang.me/about/

**Comment by imjeff** [comment]
一看就是技术迷。托管在 Github Pages 是个不错的低成本建站方案。

**Reply by yunyit** [reply]
谢谢过奖了！刚开始捣鼓，Github Pages 挺方便的。

**Comment by Liuqiran** [comment]
我最近准备将自己的网站迁到Github，这样不用管理服务器了，希望文章保存的时间更久一些，感谢分享！
